### PR TITLE
I798 chart start end dates

### PR DESCRIPTION
**Pull request by Serhiy91**

### CODE REVIEW

File: app/imports/ui/react/components/TimelineTitle.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+const enhance = compose(
+  onlyUpdateForKeys(['startDate', 'endDate', 'text']),
+  withProps(({ startDate, endDate, datum }) => ({
+    x: new Date(startDate + (endDate - startDate) / 2),
+    y: datum.y,
+  })),
+);
+
+const TimelineTitle = ({
```
Comment (Dartv) [inline app/imports/ui/react/components/TimelineTitle.js]: Why is it named `TimelineTitle` if it expects two date props? Call it something like `TimelineDateLabel`

File: app/imports/ui/react/components/TimelinePoint.js [hunk 2]
```diff
@@ -1,40 +1,102 @@
...
+    datum: {
+      x: pointDate,
+      isStart,
+      isEnd,
+      startDate,
+    },
+  }) => ({
+    dateLabel: (isStart || isEnd) && moment(pointDate).format('D MMM YYYY'),
+    pointId: `point-${id}-${index}`,
+    isPopover: !!renderPopoverContent,
```
Comment (Dartv) [inline app/imports/ui/react/components/TimelinePoint.js]: This is redundant

File: app/imports/ui/react/components/TimelineTitle.js (hunk 1 above)
Comment (Serhiy91) [reply app/imports/ui/react/components/TimelineTitle.js]: dates is for finding title position. I have remove these props in my next PR

**Comment by Serhiy91** [comment]
@Dartv fixed

**Pull request merged**